test-integration/scenarios/loans/12_liquidate_loan_invalid_liquidation_not_needed.js

```diff
@@ -36,11 +36,11 @@ module.exports = async (testContext) => {
   let initialOraclePrice;
   let collateralAmountDepositCollateral;
   if (collTokenName.toLowerCase() === "eth") {
-    initialOraclePrice = toDecimals("0.00295835", 18);
+    initialOraclePrice = "0.00295835";
```

Comment:
are we calculating the decimals in the setPrice function?

Reply:
Yes. It gets the decimals from the token contract.